tests/unit/Makefile

```diff
@@ -7,7 +7,7 @@ GTEST_DIR = ../../deps/googletest
 CPPFLAGS += -isystem $(GTEST_DIR)/include
 
 # Flags passed to the C++ compiler.
-CXXFLAGS += -g -Wall -Wextra -pthread
+CXXFLAGS += -g -Wall -Wextra -pthread -std=c++11
```

Comment:
This (or `gnu++11`) is required for declaring and defining class variables outside of a function body. I don't see any disadvantages to this, but let me know if you have thoughts!